### ISSUE TITLE
[Accessibility] There is information that is only conveyed through use of color

### ISSUE THREAD

**Issue by eileencho** [jupyter/notebook#5031]
Description: Information such as whether a notebook cell is in command mode vs. editor mode or whether a notebook is running is only conveyed by color changes. There is no other visual cue (such as text) available, so this is information that is not perceivable by users who cannot see color or color changes.

Reproduce:

1. Open a Jupyter Notebook
2. In the File directory in the Jupyter dashboard, the notebook icon has turned green to show it is running, but there is no alternative visual cue to indicate this information
3. Focus on a cell in the Notebook and toggle between Command Mode and Editor Mode. The color on the edge of the cell will change between blue and green, but no visual textual alternative appears to indicate the change in mode.

Suggestions:

- In the file directory, have text appear next to the running item that says “Running”
- For Notebook cells, have text at the selected cell that indicates Command or Editor mode

[Example from NYU Accessibility Audit Supplement](https://docs.google.com/document/d/1q_7LiqLX2WGJZ0UHNWDFafw75McZgLp9v4Ifdzn2FYE/edit#heading=h.8mvgk0mlzv3z)
[WCAG 2.0 Guidelines Section 1.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html)
[NYU Full Accessibility Audit](https://docs.google.com/spreadsheets/d/1iCiEExo4VDMOwAYtBTCdh2chyx394-1r6YwzmRILF9E/edit?usp=sharing) done with @VickySteeves

**Comment by twrobinson** [comment]
Which version of notebook are you running? In 6.0.0 neither of these issues are visible, so unless there has been a regression since 6.0.0 I think these issues are resolved:
- flashing cursor indicates edit mode
- pen icon indicated edit mode
- "Running" indicated in file browser
![Screenshot 2019-11-22 at 12 54 06](https://user-images.githubusercontent.com/5872125/69424102-15928c00-0d28-11ea-8e58-65b69e1c9bd9.png)
![Screenshot 2019-11-22 at 12 54 18](https://user-images.githubusercontent.com/5872125/69424108-175c4f80-0d28-11ea-845c-b076720385c5.png)

**Comment by eileencho** [comment]
I updated my notebook and checked again, this has indeed been resolved. Thanks for pointing it out!
